solidity/contracts/client/MailboxClient.sol

```diff
@@ -36,7 +36,7 @@ abstract contract MailboxClient is OwnableUpgradeable, PackageVersioned {
 
     IPostDispatchHook public hook;
 
-    IInterchainSecurityModule public interchainSecurityModule;
+    IInterchainSecurityModule internal _interchainSecurityModule;
```

Comment:
I think this actually might cause confusion because of the way underscores are used for parameters

Reply:
we do the same thing in `Router`